tests/php/Integration/Formatter/Command/Format/FormatCommandTest.php

```diff
@@ -24,63 +25,69 @@ public function test_good_format(): void
         $path = self::FIXTURES_DIR . 'good-format.phel';
         $oldContent = file_get_contents($path);
 
-        $command = $this->createFormatCommand();
-
-        $this->expectOutputRegex('/No files were formatted+/s');
+        $tester = new CommandTester($this->createFormatCommand());
 
         try {
-            $command->run(
-                $this->stubInput([$path]),
-                $this->stubOutput(),
-            );
+            $tester->execute(['paths' => [$path]]);
+
+            $this->assertMatchesRegularExpression('/No files were formatted+/s', $tester->getDisplay());
         } finally {
             file_put_contents($path, $oldContent);
         }
     }
 
-    public function test_bad_format(): void
+    public function test_command_uses_default_paths(): void
     {
         $path = self::FIXTURES_DIR . 'bad-format.phel';
         $oldContent = file_get_contents($path);
 
-        $command = $this->createFormatCommand();
+        Gacela::bootstrap(__DIR__, static function (GacelaConfig $config) use ($path): void {
```

Comment:
@Chemaclass does it make sense to bootstrap Gacela twice (it was already bootstrapped in the `setUpBeforeClass()`?